pkg/kube/statefulset/statefulset.go

```diff
@@ -297,3 +298,17 @@ func VolumeMountWithNameExists(mounts []corev1.VolumeMount, volumeName string) b
 	}
 	return false
 }
+
```

Comment:
we didn't need this in the `controller` package, especially since now we can export it more cleanly